gensim/test/test_sklearn_integration.py

```diff
@@ -15,12 +15,12 @@
 except ImportError:
     raise unittest.SkipTest("Test requires scikit-learn to be installed, which is not available")
 
-from gensim.sklearn_integration.sklearn_wrapper_gensim_rpmodel import SklRpModel
-from gensim.sklearn_integration.sklearn_wrapper_gensim_ldamodel import SklLdaModel
-from gensim.sklearn_integration.sklearn_wrapper_gensim_lsimodel import SklLsiModel
-from gensim.sklearn_integration.sklearn_wrapper_gensim_ldaseqmodel import SklLdaSeqModel
-from gensim.sklearn_integration.sklearn_wrapper_gensim_w2vmodel import SklW2VModel
-from gensim.sklearn_integration.sklearn_wrapper_gensim_atmodel import SklATModel
+from gensim.sklearn_integration.sklearn_wrapper_gensim_rpmodel import RpTransformer
+from gensim.sklearn_integration.sklearn_wrapper_gensim_ldamodel import LdaTransformer
+from gensim.sklearn_integration.sklearn_wrapper_gensim_lsimodel import LsiTransformer
+from gensim.sklearn_integration.sklearn_wrapper_gensim_ldaseqmodel import LdaSeqTransformer
+from gensim.sklearn_integration.sklearn_wrapper_gensim_w2vmodel import W2VTransformer
+from gensim.sklearn_integration.sklearn_wrapper_gensim_atmodel import AuthorTopicTransformer
```

Comment:
As long as we're making breaking renames/refactors for clarity/concision, these package-names have a lot of redundancy, too. `gensim.sklearn_integration.sklearn_wrapper_gensim_atmodel` repeats both `sklearn` and `gensim`, and `integration` and `wrapper` have overlapping meanings. 

So perhaps just: `gensim.sklearn_wrappers.atmodel`? And that's if `at` is already easily recognized from use elsewhere as abbreviation; otherwise `gensim.sklearn_wrappers.authortopicmodel` or even `gensim.sklearn_wrappers.authortopic`?

Or maybe even:, given that these wrappers are each pretty slight-in-size, they could either go: (1) in the respective associated model file – that is W2VTransformer goes into `gensim.models.word2vec` (and probably loses its abbreviation, to match the non-abbreviation used in that file's classes); or (2) into a single `sklearn_wrappers` file, alongside the `BaseSklearnWrapper`?

(And regarding `BaseSklearnWrapper` - not sure it adds much beyond alternative of using skearn's own BaseEstimator and TransformerMixin. It forces more rigor in overriding the necessary abstract-methods, which sklearn itself never does, but has less functionality in `set_params()` and other aspects of introspectability.)